internal/cmd/broker/delete_broker.go

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2018 The Service Manager Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package broker
+
+import (
+	"fmt"
+
+	"github.com/Peripli/service-manager-cli/internal/output"
+	"github.com/Peripli/service-manager-cli/internal/util"
+
+	"github.com/spf13/cobra"
+
+	"github.com/Peripli/service-manager-cli/internal/cmd"
+	"github.com/Peripli/service-manager-cli/pkg/smclient"
+)
+
+// DeleteBrokerCmd wraps the smctl list-brokers command
+type DeleteBrokerCmd struct {
+	*cmd.Context
+
+	names []string
+}
+
+// NewDeleteBrokerCmd returns new list-brokers command with context
+func NewDeleteBrokerCmd(context *cmd.Context) *DeleteBrokerCmd {
+	return &DeleteBrokerCmd{Context: context}
+}
+
+func (dbc *DeleteBrokerCmd) buildCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:     "delete-broker [name] <name2 <name3> ... <nameN>>",
+		Aliases: []string{"db"},
+		Short:   "Deletes brokers",
+		Long:    `Delete one or more brokers with name.`,
+		PreRunE: cmd.PreRunE(dbc, dbc.Context),
+		RunE:    cmd.RunE(dbc),
+	}
+}
+
+// Validate validates command's arguments
+func (dbc *DeleteBrokerCmd) Validate(args []string) error {
+	if len(args) < 1 {
```

Comment:
They can be if you write something like this:

```sh
smctl db "" "" ""
```

Not sure if we have to care about this, because Service Manager will reject them anyway as not found brokers

Reply:
We should avoid bombarding the service manager with requests that we know are invalid